WriteBack/ListTriggerAsync.cs

```diff
@@ -0,0 +1,60 @@
+﻿using Microsoft.Extensions.Logging;
+using System;
+using System.Threading.Tasks;
+using Microsoft.Azure.Cosmos;
+using System.Collections.Generic;
+using StackExchange.Redis;
+using System.Linq;
+using Microsoft.Azure.Cosmos.Linq;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Redis.Samples
+{
+    public static class WriteBack
+    {
+        //Redis Cache primary connection string from local.settings.json
+        public const string localhostSetting = "redisLocalhost";
+        private static readonly IDatabase cache = ConnectionMultiplexer.Connect(Environment.GetEnvironmentVariable(localhostSetting)).GetDatabase();
+
+        //CosmosDB Endpoint from local.settings.json
+        public const string Endpoint = "Endpoint";
+
+        //CosmosDB database name and container name declared here
+        public const string databaseName = "back";
+        public const string containerName = "async";
+
+        //Uses the key of the user's choice and should be changed accordingly
+        public const string key = "listTest2";
```

Comment:
nit: we should not be naming the list "test" maybe a more descriptive name would work.